contracts/AirdropCampaignData.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+/// @title AirdropCampaignData - Data contract for storing of daily merkleRootHashes of airbro Campaigns
+contract AirdropCampaignData {
+    address public admin = 0xF4b5bFB92dD4E6D529476bCab28A65bb6B32EFb3;
+    uint16 public batchHashArrayLimit = 600;
+    uint16 public batchChainArrayLimit = 600;
+
+    enum Chains {
+        Zero,
+        Eth,
+        Pols
+    }
+
+    struct AirdropData {
+        Chains chain;
+        bytes32[] hashArray;
+        bool airdropFinished;
+    }
+
+    mapping(address => AirdropData) public airdrops;
```

Comment:
Note: Smart contracts can have the same address on different chains. (CREATE2) With current implementation, only one chain could get the drop.

Reply:
Not sure If I understand this

Reply:
> Not sure If I understand this.
> 
> EDIT: I see, so we can use the [CREATE2](https://github.com/thegostep/solidity-create2-deployer) to deploy on two chains with the same address. 

This means that other people can do the same thing as well. 
This means that you could reward same address only once (on one chain)
Maybe i misunderstood, but what is the point of having `enum Chains {
        Zero,
        Eth,
        Pols
    }`

Reply:
This okay since we're using one chain per contract